linear_operator/operators/_linear_operator.py

```diff
@@ -2013,7 +2042,7 @@ def representation(self) -> Tuple[torch.Tensor, ...]:
         Returns the Tensors that are used to define the LinearOperator
         """
         representation = []
-        for arg in self._args:
+        for arg in list(self._args) + list(self._differentiable_kwarg_vals):
```

Comment:
Can be agnostic to the types here if you just want to iterate
```suggestion
        for arg in itertools.chain(self._args, self._differentiable_kwarg_vals):
```